scripts/makefile/newrelic.sh

```diff
@@ -3,7 +3,7 @@
 # installs newrelic.com agent extension
 
 # use `php -i | grep "additional .ini"` to get it
-PHP_INI_DIR=/etc/php7/conf.d
+PHP_INI_DIR=/etc/php8/conf.d
```

Comment:
it could use `.env.`setting to reuse in docker configuration

Reply:
Keeping the variable at a single place would be best indeed
However I have the feeling that we twisted the purpose of `.env` file and mixed 2 different purpose :
- local overrides
- shared variables
 
Maybe they should be 2 different files and only one "local overrides" file would use the "non-versioned" `.default` trick. 
Because at the moment, it often happens that an outdated `.env` file breaks build and I think we can avoid that easily

See what I mean ?

Reply:
We could have a global environment variable `PHP_VERSION`

it's remains removed in https://github.com/skilld-labs/skilld-docker-container/commit/60ec7768190cdb8c8f0a0e495d3d7424db479358